### PR TITLE
[ISSUE/DESIGN/CONTACT] Fix /contact design

### PR DESCRIPTION
**Pull request by Kazouskyn**
Work to fix Issue #12  [DESIGN/CONTACT] Fix /contact design

### CODE REVIEW

File: pages/contact/Contact.module.css [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-    margin-top: 50px;
+    margin-top: 60px;
     margin-bottom: 50px;
     display: flex;
     justify-content: flex-start;
     align-items: center;
     flex-direction: column;
+    min-height: calc(100vh - 300px);
+    background: rgb(51,51,51);
+    background: linear-gradient(0deg, rgba(51,51,51,1) 0%, rgba(16,16,16,1) 75%);
```
Comment (TeflonTrout) [inline pages/contact/Contact.module.css:10]: Let's change this to a more symmetrical gradient. It doesn't display too well on my 13" display. `linear-gradient(0deg, rgba(51,51,51,1) 33%, rgba(16,16,16,1) 66%); ` should look nice.

File: pages/contact/index.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ const index = () => {
         <h1>Contact</h1>
         <div className={styles.hero}>
           <div className={styles.container}>
-            <h1>Email Us: *insert email here*</h1>
+            <h1>Email Us: <a href='mailto:20dollarinsuranceproduction@gmail.com'>20DollarInsuranceProduction@gmail.com</a></h1>
```
Comment (TeflonTrout) [inline pages/contact/index.tsx:10]: Didn't realize the email address was that bulky. How about changing to just an `h1` tag saying `Email Us` that links to our email.

**Review by TeflonTrout — CHANGES_REQUESTED** [review]
The new footer and `/contact` page look great. Just some small design changes before merging.

File: pages/contact/index.tsx (hunk 2 above)
Comment (Kazouskyn) [reply pages/contact/index.tsx]: fixed in the latest push: fix pr comment to put only email us on contact page

File: pages/contact/Contact.module.css (hunk 1 above)
Comment (Kazouskyn) [reply pages/contact/Contact.module.css]: fixed in push: fix gradient

**Comment by TeflonTrout** [comment]
Closes #28

**Pull request merged**